### PR TITLE
Add serializer for Shell Notebooks with file ending .shell-notebook

### PR DESCRIPTION
**Pull request by tanhakabir**

### CODE REVIEW

File: src/NotebookSerializer.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+				kind: cell.kind,
+				language: cell.languageId,
+				value: cell.value
+			});
+		}
+
+        // Give a string of all the data to save and VS Code will handle the rest 
+		return new TextEncoder().encode(JSON.stringify(contents));
+    }    
+}
```
Comment (TylerLeonhardt) [inline src/NotebookSerializer.ts]:
```suggestion
}

```
nit newline at EOF :)

**Review by TylerLeonhardt — APPROVED** [review]
LGTM :)

**Comment by TylerLeonhardt** [comment]
@tanhakabir invited you as a collaborator so feel free to merge this in when you're ready

**Comment by tanhakabir** [comment]
> @tanhakabir invited you as a collaborator so feel free to merge this in when you're ready

whoops didn't see this invite yet, one sec!

**Comment by TylerLeonhardt** [comment]
I literally just did it :) no worries

**Pull request merged**